### ISSUE TITLE
!include_dir_list is not correctly parsed for automations. 

### ISSUE THREAD

**Issue by bavo** [keesschollaart81/vscode-home-assistant#395]
The plugin creates errors for each of my automations listed in the `!include_dir_list folder. `

It looks like the plugin can only handle `!include_dir_list_merged` types of configuration.

Example:

#### **`automation.yaml`**
``` yaml
automation: !include_dir_list automations/
```

#### **`automations/bureau/lamp-aan.yaml`**
``` yaml
alias: "Bureau: Bureau lamp aan"
trigger:
  - platform: event
    event_type: deconz_event
    event_data: 
      id: schakelaar_bureau
      event: 1000
action:
  service: light.turn_on
  data:
    entity_id: light.bureau 
```

#### **`VsCode example`**
<img width="523" alt="image" src="https://user-images.githubusercontent.com/1813273/87684190-3e1d3980-c782-11ea-8bc2-cbf27fbfc99c.png">

**Comment by frenck** [comment]
Which version of the extension are you on?

**Comment by bavo** [comment]
The problem seems to be located in the `!include_dir_list` and not within the automation part of the code. 
I can reproduce this problem if I split my `scenes` up into different files as well.

I'll try and search for the problem a bit more

**Comment by CastleGreyscale** [comment]
I also have this problem, but it happens for both **automations** and **scenes** both using !include_dir_list. However I also use !include_dir_list for **light** and **sensor** with no errors.

**Comment by frenck** [comment]
@CastleGreyscale The problem is clear to me, I hope I can patch it up today.

**Comment by frenck** [comment]
Fixed in #410
